### PR TITLE
Service Delivery Report aggregation

### PR DESCRIPTION
**Pull request by Rohit25negi**
Planning to run it in parallel to ls_tasks because I think it will not be very good to run it in parallel to agg_Awc which is already a heavy step.

### CODE REVIEW

**Comment by Rohit25negi** [comment]
@calellowitz  can you see this Prs?

**Comment by Rohit25negi** [comment]
> All agg steps should be run as soon as they can. If this can be run in parallel to agg_awc it should be moved up there (furthermore agg_awc isn't that heavy anymore since some fixes we made).

I see, Ok then  I dont see any problem in moving this backward. Thanks of input. Much appreciated

**Comment by Rohit25negi** [comment]
@calellowitz  does it look fine to you?

**Comment by Rohit25negi** [comment]
gentle reminder on this @calellowitz

File: dags/dashboard_subdags.py [hunk 1]
```diff
@@ -184,6 +191,9 @@ def generate_uuid():
     agg_child_health_temp >> agg_awc_table
     child_health_monthly >> agg_awc_table
     agg_ccs_record >> agg_awc_table
+    agg_child_health_temp >> update_service_delivery_report
+    child_health_monthly >> update_service_delivery_report
```
Comment (calellowitz) [inline dags/dashboard_subdags.py]: This line isnt really necessary since `agg_child_health_temp` already depends on `child_health_monthly`

**Comment by Rohit25negi** [comment]
@calellowitz  can you suggest something how we can test it when we can't deploy this to india

**Comment by calellowitz** [comment]
You should be able to run it locally. I usually do it for the same months as our tests, and comment out the agg part of the test setup. That way all the comparisons run but against the airflow generated data

**Comment by Rohit25negi** [comment]
tested on india

**Pull request merged**